src/db-api.js

```diff
@@ -216,17 +201,29 @@ const getTxsInputs = (db: Pool) => async (txIds: Array<number>)
 * @param {Db Object} db
 * @param {Array<Transaction>} txIds
 */
-const getTxsOutputs = (db: Pool) => async (txIds: Array<number>)
-: Promise<TypedResultSet<TxOutput>> =>
+const getTxsOutputs = (db: Pool) => async (txIds: Array<string>)
+: Promise<TypedResultSet<TxOutputDbResult>> =>
   (db.query({
     text: `SELECT
-      tx.id as "txDbId", tx_out.address, tx_out.value, tx_out.index
+        tx.id as "txDbId", tx_out.address, tx_out.value, tx_out.index,
+        tx_out.id AS "txOutDbId"
       FROM tx 
       INNER JOIN tx_out ON tx.id = tx_out.tx_id
       WHERE tx.id = ANY($1)`,
     values: [txIds],
   }): any)
 
+const getTokens = (db: Pool) => async (txOutIds: Array<string>)
+: Promise<TypedResultSet<TokenWithDbId>> =>
+  (db.query({
+    text: `SELECT
+        RIGHT(ma.policy::text, -2) AS "policyId", RIGHT(ma.name::text, -2) AS "assetName",
+        ma.quantity, ma.tx_out_id AS "txOutDbId"
+      FROM ma_tx_out ma 
+      WHERE ma.tx_out_id = ANY($1)`,
```

Comment:
are we sure the db has an index on tx_out_id column?

Reply:
```
Indexes:
    "ma_tx_out_pkey" PRIMARY KEY, btree (id)
    "unique_ma_tx_out" UNIQUE CONSTRAINT, btree (policy, name, tx_out_id)
    "idx_ma_tx_out_tx_out_id" btree (tx_out_id)
```